autovalet_system/scripts/state_machine.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+sim=$1
+delay=20
+echo "Waiting $delay seconds for system to boot up"
```

Comment:
so this is a guess. there's no way i can programatically tell that gazebo/rviz/rtab have all finished initializing. and i wanted everything to run with one command. so i wait 20 seconds before starting the state machine assuming that everything should be up by then, which it is. i'm open to other ways to doing this, but it's the best i could come up with. @SubramanianKrish @evamo0508 , thoughts?

Reply:
I think it's fine since what matters is it runs smoothly after booting up. If by trial and error we need 20 secs, let it be. but if maybe 15 still works, I would agree with lowering the value down a bit. I am fine with delaying the program with a fixed time.

Reply:
I'd say start at 10sec and move up or down trying to reduce this time delay. As low as we can get for a perfectly running system.